src/dotnet/commands/dotnet-new/CSharp_Console1.1/Program.cs

```diff
@@ -1,9 +1,12 @@
 using System;
 
-class Program
+namespace ConsoleApp
```

Comment:
Yes, because of https://github.com/dotnet/sdk/issues/271. All item templates generally have the namespace and while this makes hello world simpler - it adds complexity to work with real projects.